src/Console/ModelsCommand.php

```diff
@@ -36,6 +36,20 @@
  */
 class ModelsCommand extends Command
 {
+    protected const RELATION_TYPES = array(
+        'hasMany' => '\Illuminate\Database\Eloquent\Relations\HasMany',
+        'hasManyThrough' => '\Illuminate\Database\Eloquent\Relations\HasManyThrough',
+        'hasOneThrough' => '\Illuminate\Database\Eloquent\Relations\HasOneThrough',
+        'belongsToMany' => '\Illuminate\Database\Eloquent\Relations\BelongsToMany',
+        'hasOne' => '\Illuminate\Database\Eloquent\Relations\HasOne',
+        'belongsTo' => '\Illuminate\Database\Eloquent\Relations\BelongsTo',
+        'morphOne' => '\Illuminate\Database\Eloquent\Relations\MorphOne',
+        'morphTo' => '\Illuminate\Database\Eloquent\Relations\MorphTo',
+        'morphMany' => '\Illuminate\Database\Eloquent\Relations\MorphMany',
+        'morphToMany' => '\Illuminate\Database\Eloquent\Relations\MorphToMany',
+        'morphedByMany' => '\Illuminate\Database\Eloquent\Relations\MorphToMany'
```

Comment:
I suggest to start using the `:class` syntax here then, now that the code is already touched

Reply:
Has been resolved in https://github.com/barryvdh/laravel-ide-helper/pull/987/commits/392de13a19d6b4d87e9d0f8cf08d51f5a59ff128